schema/codelists/tenderStatus.csv

```diff
@@ -1,8 +1,8 @@
-Code,Title,Description
-planning,Planning,"A future contracting process is being considered. Early information about the process can be provided in the tender section. A process with this status might provide information on early engagement or consultation opportunities, during which the details of a subsequent tender can be shaped."
-planned,Planned,"A contracting process is scheduled, but is not yet taking place. Details of the anticipated dates can be provided in the tender block."
-active,Active,A tender process is currently taking place.
-cancelled,Cancelled,The tender process has been cancelled.
-unsuccessful,Unsuccessful,The tender process was unsuccessful.
-complete,Complete,The tender process is complete.
-withdrawn,Withdrawn,No further information on this process is available under this ocid.
+Code,Title,Description,Deprecated,Deprecation note
+planning,Planning,"A future contracting process is being considered. Early information about the process can be provided in the tender section. A process with this status might provide information on early engagement or consultation opportunities, during which the details of a subsequent tender can be shaped.",1.2,"This code has been deprecated, because it is unnecessary since contracting processes are split from planning processes (which do not have any states themselves)."
+planned,Planned,"The contracting process is scheduled, but contracting documents (for example, planning documents) are not yet available to potential suppliers.",1.2,Deprecated because the same information can be determined by comparing `tender/datePublished` (new in 1.2) to `date` (or its absence)
```

Comment:
```suggestion
planned,Planned,"The contracting process is scheduled, but contracting documents (for example, planning documents) are not yet available to potential suppliers.",1.2,Deprecated because the same information can be determined by the absence of `tender/datePublished` (new in 1.2) or by comparing it to `date`.
```

Reply:
I find the example of planning documents to be a bit confusing here since my understanding is that `tender.status` should only be published as part of a contracting process. Could it instead be replaced with bidding documents (to align with the document type codelist)?

Regarding the deprecation notice, my understanding is that `tender.datePublished` is only to be used retrospectively and that future notice dates should be communicated using `tender.communication.futureNoticeDate` so I'm struggling to understand how comparing `tender/datePublished` to `date` could be used to determine this state.

Reply:
I'm not sure why that was used in e33c21c0b255bc337879c92a63028f03f00435aa but we can change to "procurement documents" like in the 'active' code.

For deprecation, we can just leave it as the absence of `tender.datePublished`. There are certainly many dates that would mean the tender is active.

Reply:
Sounds good. I've made those changes in f6217cc57e6c6ab6121d8f681d29366e1bd86ced

Reply:
34ee1350796350408d617c0bc89739949fd1f78e